torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -806,14 +806,19 @@ def get_attr(
         ],
         fx_graph_module: torch.fx.GraphModule,
     ):
+        # TODO: Constant tensors and buffer/weights are both categorized into `get_attr`,
+        # but they are different to ONNX. We need to distinguish them.
+        # Constant tensors are inputs of the graph, and buffer/weights are initializers.
```

Comment:
```suggestion
        # Constant tensors should become ONNX constants in the graph, while buffers/weights ONNX initializers.
        # For now they are all converted to ONNX initializers.
```
Is this more accurate?